front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarPhases.tsx

```diff
@@ -11,48 +11,50 @@ import messages from '../../messages';
 import FilterSidebarPhasesItem from './FilterSidebarPhasesItem';
 
 type Props = {
-  phases?: IPhaseData[];
-  selectedPhase?: string | null;
-  onChangePhaseFilter?: (phase: string | null) => void;
+  phases: IPhaseData[];
+  selectedPhase: string | undefined;
```

Comment:
This is never null in the two places where we pass `selectedPhase` (via the input/proposals manager).

Reply:
It should also not be optional in a phases filter.